network-api/networkapi/urls.py

```diff
@@ -40,6 +40,7 @@
         name='how-do-i-wagtail'
     ),
     url(r'^cms/', include(wagtailadmin_urls)),
+    # FIXME:TODO: what to do with this /en route
```

Comment:
What to do with line 44? Do we keep the whole line? Remove the whole line? Replace `en` with the locale variable?

Reply:
we want to keep it: the `/cms` is not part of the localised routes, but everyone in the engineering team keeps accidentally going to `localhost:8000/en/cms` etc, so we added this route to make sure that even though that's not a real route as far as wagtail is concerned, _we_ make it a real route and you can get to the cms rather than hitting a 404 =)